cfgov/v1/preprocessed/js/modules/classes/BreakpointHandler.js

```diff
@@ -72,7 +70,7 @@ function _init( self ) {
  */
 function watchWindowResize() {
   var self = this;
-  $( window ).bind( 'resize', function() {
+  window.addEventListener( 'resize', function() {
```

Comment:
Add `require( './polyfill/event-listener.js' );` at top of module for IE8 support.